modules/nf-core/merquryfk/merquryfk/main.nf

```diff
@@ -37,14 +45,18 @@ process MERQURYFK_MERQURYFK {
     }
     def args = task.ext.args ?: ''
     prefix = task.ext.prefix ?: "${meta.id}"
+    def mat_ktab = matktab ? "${matktab.find{ it.toString().endsWith(".ktab") }}" : ''
```

Comment:
Does this need the `find`? Is it not just one file?

Reply:
yes, merquryfk -t generates a collection of files.

Reply:
Right, that is weird, but I see that is what the fastk tool is doing.